src/nostr/index.ts

```diff
@@ -28,6 +28,19 @@ export const fetchUserProfile = async (pubkey: string, pool: SimplePool) => {
   return result;
 };
 
+export async function fetchFollows(pubkey: string, pool: SimplePool) {
```

Comment:
Why not use already existing ListProvider? Where kind 3s are already fetched? Wouldn't this be unnecessary network calls?